conformance/submission_details_template.txt

```diff
@@ -58,10 +58,12 @@ Target SYCL device:
 # and device combinations.
 #
 SYCL implementation version(s):
+Device(s):
+Platform:
```

Comment:
```suggestion
Platform(s):
```

Reply:
Minor tweak, I'd add a `(s)` as the process allows for a submission which supports other platforms//backends.